tests/components/august/test_binary_sensor.py

```diff
@@ -1 +1,71 @@
 """The binary_sensor tests for the august platform."""
+
+import pytest
+
+from homeassistant.components.lock import DOMAIN as LOCK_DOMAIN
+from homeassistant.const import (
+    ATTR_ENTITY_ID,
+    SERVICE_LOCK,
+    SERVICE_UNLOCK,
+    STATE_OFF,
+    STATE_ON,
+)
+
+from tests.components.august.mocks import (
+    _create_august_with_devices,
+    _mock_doorbell_from_fixture,
+    _mock_lock_from_fixture,
+)
+
+
+@pytest.mark.skip(
+    reason="The lock and doorsense can get out of sync due to update intervals, "
+    + "this is an existing bug which will be fixed with dispatcher events to tell "
+    + "all linked devices to update."
+)
+async def test_doorsense(hass):
+    """Test creation of a lock with doorsense and bridge."""
+    lock_one = await _mock_lock_from_fixture(
+        hass, "get_lock.online_with_doorsense.json"
+    )
+    lock_details = [lock_one]
+    await _create_august_with_devices(hass, lock_details)
+
+    binary_sensor_abc_name = hass.states.get("binary_sensor.abc_name_open")
+    assert binary_sensor_abc_name.state == STATE_ON
+
+    data = {}
+    data[ATTR_ENTITY_ID] = "lock.abc_name"
+    assert await hass.services.async_call(
+        LOCK_DOMAIN, SERVICE_UNLOCK, data, blocking=True
+    )
+
+    binary_sensor_abc_name = hass.states.get("binary_sensor.abc_name_open")
+    assert binary_sensor_abc_name.state == STATE_ON
+
+    assert await hass.services.async_call(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5054188)